python/test/eager_mode/annotate_args.py

```diff
@@ -21,7 +21,8 @@
 # CHECK: Torch Tensor (shape=(1, 3, 32, 32), dtype=torch.float32)
 # -----
 # CHECK: PASS - simple
```

Comment:
I think you have to change this line to XFAIL too

Reply:
Yes that's correct (and a weakness of this testing harness that I should think about).